.github/workflows/test-integration.yml

```
@@ -0,0 +1,73 @@
+name: Integration
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '**.md'
```

Does this makes sense instead of the `technote-space/get-diff-action` step patterns? Maybe using the paths here will make sure that the CI workflow is triggered only when any of those files are changed, so the diff step wouldn't be needed, theoretically :)

@mazzy89 

```suggestion
    paths:
      - '**.go'
      - '**/go.mod'
      - '**/go.sum'
      - '**/testdata/**'
```

---

I usually use `get-diff-action` when I want to make some steps conditional according some files changes.

In this case it makes sense to use '**.md' but as always it's a trade off. If let's say tomorrow we add another file that it should not trigger any related integration tests then we need to remember to add it here. while the `paths` is in my opinion better because it explicit communicate in this case when we want to run tests due to changes on the go files. ultimately I would vote for this case to `paths` to clearly state what files have to be changed to trigger tests.